.github/pull_request_template.md

```diff
@@ -0,0 +1,32 @@
+##### SUMMARY
+<!--- Describe the change below, including rationale and design decisions -->
+
+<!--- HINT: Include "Fixes #nnn" if you are fixing an existing issue -->
+
+<!--- Please do not forget to include a changelog fragment:
+      https://docs.ansible.com/ansible/devel/community/collection_development_process.html#creating-a-changelog-fragment
+      No need to include one for docs-only or test-only PR, and for new plugin/module PRs.
+      Read about more details in CONTRIBUTING.md.
+      -->
+
+##### ISSUE TYPE
+<!--- Pick one or more below and delete the rest.
+      'Test Pull Request' is for PRs that add/extend tests without code changes. -->
+- Bugfix Pull Request
+- Docs Pull Request
+- Feature Pull Request
+- New Module/Plugin Pull Request
+- Refactoring Pull Request
+- Test Pull Request
+
```

Comment:
It occurred to me: Deprecation. It is something that we usually do not want to be mixed with any other type of updates, I think that makes it worth of its own issue type.

Reply:
I'm wondering whether it is specialized/common enough to warrant its own category in this list (it could also be filed under Feature Pull Request). I'm completely unsure right now.

Reply:
In that case, I'd say let's go without it. Later on it is easier to change our minds by adding something than it is by removing something that people is already using.